repos/system_upgrade/common/models/unsafepythonpaths.py

```diff
@@ -0,0 +1,22 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class UnsafePythonPaths(Model):
+    """
+    Information about third-party Python modules found on system.
+
+    These modules may interfere with the upgrade process or introduce unexpected
+    behavior during runtime or after reboot.
+    """
+    topic = SystemInfoTopic
+
+    is_third_party_module_present = fields.Boolean(default=False)
```

Comment:
it's a question whether it has a value if third party python modules are listed. but currently they are not listed in the model et